### DISCUSSION TITLE
Can install couchdb for current user only on Windows ?

### DISCUSSION THREAD

**Discussion by topapp2030** [apache/couchdb#3606] [Q&A] [answered]
hi ,
Can install couchdb for current user only on Windows ?

**Comment by wohali** [comment, marked as answer]
No, this is not currently supported in the automated installer. Administrative rights are required for install.

However, you can disable the service and start up CouchDB from the Command Line when you need it. I am aware of people in the world using CouchDB on Windows with multiple users, each with their own databases.
